### PR TITLE
Updated ReleaseProcess.md

### PR DESCRIPTION
**Pull request by matsduf**
* GUI does not use Perl. Updated description for GUI.
* Description of how to do the release is updated.

### CODE REVIEW

**Comment by matsduf** [comment]
@pamasse, I really want you to look at this.

File: docs/internal-documentation/maintenance/ReleaseProcess.md [hunk 1]
```diff
@@ -20,8 +20,20 @@ The version numbers can be found in these Perl modules:
...
+
  * zonemaster-gui - [GUI.pm](https://github.com/zonemaster/zonemaster-gui/blob/master/lib/Zonemaster/GUI.pm)
 
+ * zonemaster-gui - [Installation.md](https://github.com/zonemaster/zonemaster-gui/blob/master/docs/Installation.md):
+   The version is part of the download path (a diretory). It can be repeated several times.
+ * zonemaster-gui - [package.json](https://github.com/zonemaster/zonemaster-gui/blob/master/package.json):
+   In the top of the file, the version is given after "version".
+ * zonemaster-gui - [environment.prod.ts](https://github.com/zonemaster/zonemaster-gui/blob/master/src/environments/environment.prod.ts):
+   Under "client-info", "version" should point at the version number.
+ * zonemaster-gui - [environment.ts](https://github.com/zonemaster/zonemaster-gui/blob/master/src/environments/environment.ts):
```
Comment (pamasse) [inline docs/internal-documentation/maintenance/ReleaseProcess.md]: Only used for development purpose, but yes, for a new version, we should update this "version" number too.

File: docs/internal-documentation/maintenance/ReleaseProcess.md [hunk 2]
```diff
@@ -92,6 +109,8 @@ MANIFEST.SKIP, i.e. no missing or extra files:
 
 ## 7. Produce distribution tarballs
 
+This section is not relevant for Zonemaster-GUI.
```
Comment (pamasse) [inline docs/internal-documentation/maintenance/ReleaseProcess.md]: Yes, we have a distribution tarballs. 

The requirements are nodejs and npm. There are available from the [official website]( https://nodejs.org/en/).
Minimal version of Nodejs is 6.0 but install the last LTS version available.

To build the tarballs, steps are: 

1. `git clone https://github.com/zonemaster/zonemaster-gui.git`  
2. `cd zonemaster-gui`
3. `npm install` 
4. `npm run release`

The distribution tarballs is in the root level of the zonemaster-gui folder. Its name is `zonemaster_web_gui.zip`.

File: docs/internal-documentation/maintenance/ReleaseProcess.md [hunk 3]
```diff
@@ -142,49 +166,19 @@ Write a detailed instruction with commands.
...
-
-    git tag v1.0.0
-	git push origin --tags
-
-### ToDo
-
-Write a description how to set release in Github to get a nice presentation.
+For each repository, go to "releases" in Github and select "draft a new release".
+Use the version number as tag and create a new release description in the same
+format as previous releases.
```
Comment (pamasse) [inline docs/internal-documentation/maintenance/ReleaseProcess.md]: For zonemaster-gui, add the distributed tarballs as attached file of the new release.

File: docs/internal-documentation/maintenance/ReleaseProcess.md (hunk 2 above)
Comment (matsduf) [reply docs/internal-documentation/maintenance/ReleaseProcess.md]: Fixed.

File: docs/internal-documentation/maintenance/ReleaseProcess.md (hunk 3 above)
Comment (matsduf) [reply docs/internal-documentation/maintenance/ReleaseProcess.md]: Fixed.

**Comment by matsduf** [comment]
@pamasse, please review again after my updates.

**Comment by sandoche2k** [comment]
@matsduf can this be merged?

**Pull request merged**